dhlab/api/dhlab_api.py

```diff
@@ -93,7 +93,7 @@ def get_places(urn: str) -> DataFrame:
 
     :param str urn: uniform resource name, example: ``URN:NBN:no-nb_digibok_2011051112001``
     """
-    params = locals()
+    params = {"urn": urn}
     r = requests.post(f"{BASE_URL}/places", json=params)
```

Comment:
```suggestion
    r = requests.post(f"{BASE_URL}/places", json={"urn": urn})
```
We can probably just make the `params`-dictionary inline since it's only the one argument and it's not used elsewhere 🙂